### DISCUSSION TITLE
Lxqt

### DISCUSSION THREAD

**Discussion by praveenkumar222** [Yisus7u7/termux-desktop-lxqt#10] [General]
Your lxqt is so beautiful and i like this version icons .
But change the all wallpaper and kept in good manner.

**Comment by JesusChapman** [comment]
> Your lxqt is so beautiful and i like this version icons .
> But change the all wallpaper and kept in good manner.

Unfortunately I do not have much time for this project, but if you want to correct errors and help you can do so, I will gladly accept your extraction request.

**Comment by justondean** [comment]
Hey, This is amazing work. Thank you so much for these scripts!
Do you know if it's possible to install multiple desktops? I'm ran this and got lxqt working, but can i go back and add xfce to it?

Can i "uninstall" this easily someway? I was thinking of switch my table to the the xfce desktop to try it.
